README.md

```diff
@@ -66,13 +66,13 @@ the platform.
 A Dockerfile is provided in `./cross_build` that demonstrates how to cross compile the example application and all dependencies. The example build is for `aarch64`. To adopt to a custom toolchain, adjust the Dockerfile to include and use the custom toolchain. Assuming the custom toolchain is available in "camera-toolchain.tar.gz", it can be installed it into the image by modifying the Dockerfile as follows:
```

Comment:
This states the example is for aarch64 but the example is clearly armv4

Reply:
The default example dockerfile was for aarch64, the adaptations in the readme was for showing how to adapt to armv4 (the sigmastar example). Trying to clarify a bit.